### DISCUSSION TITLE
any estimate on the next release to solve the reloader issue?

### DISCUSSION THREAD

**Discussion by ntextreme3** [pallets/werkzeug#2628] [Q&A] [answered]
https://github.com/pallets/werkzeug/pull/2604 was merged but not released. Any estimate on when that would be released? Thanks!

**Comment by davidism** [comment, marked as answer]
We don't keep a strict release cadence. Releases happen when a maintainer has enough time to decide that the release is ready and to make it happen.

**Reply by ntextreme3** [reply]
thanks!
